opm/polymer/fullyimplicit/BlackoilPolymerModel.hpp

```diff
@@ -67,6 +67,7 @@ namespace Opm {
         /// \param[in] has_vapoil       turn on vaporized oil feature
         /// \param[in] has_polymer      turn on polymer feature
         /// \param[in] has_plyshlog     true when PLYSHLOG keyword available
+        /// \param[in] has_shrate       true when PLYSHLOG keyword available
```

Comment:
There are multiple non-documented parameters here: some that should have been documented previously, and the new wells_bore_diameter. Please document all public interfaces.